handlers/status.py

```diff
@@ -1,97 +1,107 @@
-        f = open(STATUS_DB, "w")
-        f.write(json.dumps({}))
-        f.close()
-    with open(STATUS_DB, "r") as f:
-        status_content = json.load(f)
-        if status_content.get(group_id) and len(status_content[group_id]) != 0:
-            for nickname in status_content[group_id]:
-                listed_status_users += f"\n{nickname}, {status_content[group_id][nickname]}"
-            listed_status_users = listed_status_users[:-1]
-        else:
-            listed_status_users += " No users have been set."
-    update.message.reply_text(listed_status_users)
+    with psycopg2.connect(dbname=url.path[1:],user=url.username,password=url.password,host=url.hostname,port=url.port) as conn:
+        listed_status_users = "The users in the status list are:"
+        group_id = str(update.effective_chat.id)
+        with conn.cursor() as cursor:
+            cursor.execute('SELECT name, xuid FROM status WHERE id = {}'.format(group_id))
+            curr_record = cursor.fetchone()
+            if not curr_record or not curr_record[0]:
+                listed_status_users += " No users have been set."
+            else:
+                for x in range(len(curr_record[0])):
+                    listed_status_users += f"\n- {curr_record[0][x]}, {curr_record[1][x]}"  
+        update.message.reply_text(listed_status_users)
 
 
 def status(update, context):
-    players = []
-    group_id = str(update.effective_chat.id)
-    if not os.path.isfile(STATUS_DB):
-        f = open(STATUS_DB, "w")
-        f.write(json.dumps({}))
-        f.close()
-    with open(STATUS_DB, "r") as f:
-        status_content = json.load(f)
-        if status_content.get(group_id) and len(status_content[group_id]) != 0:
-            for name in status_content[group_id]:
-                try:
-                    user = XboxApi.get_player(status_content[group_id][name], name)
-                    players.append(str(user))
-                except requests.HTTPError:
-                    update.message.reply_text("Something went wrong with the API.")
-            players.sort()
+    with psycopg2.connect(dbname=url.path[1:],user=url.username,password=url.password,host=url.hostname,port=url.port) as conn:
+        players = []
+        group_id = str(update.effective_chat.id)
+        with conn.cursor() as cursor:
+            cursor.execute('SELECT name, xuid FROM status WHERE id = {}'.format(group_id))
+            curr_record = cursor.fetchone()
+            if not curr_record or not curr_record[0]:
+                update.message.reply_text("No users have been set to show status of.")
+            else:
+                for x in range(len(curr_record[0])):
+                    try:
+                        user = XboxApi.get_player(curr_record[1][x], curr_record[0][x])
+                        players.append(str(user))
+                    except requests.HTTPError:
+                        update.message.reply_text("Something went wrong with the API.")
```

Comment:
You can simplify this for loop by using zip. For example:

for (name, id) in zip(curr_record):
    user = XboxApi.get_player(id, name)